### PR TITLE
Nav-Bar-Fix .#1

### PR DESCRIPTION
**Pull request by yusufroqib**
This is a fix for Nav bar issue .#1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,6 +6,9 @@
     <title>Document</title>
 </head>
 <body>
+    <header>
+        <nav> Nav Bar Test </nav>
```
Comment (DevBigEazi) [inline index.html:10]: Pls capitalize the text.Thanks

File: index.html [hunk 2]
```diff
@@ -6,6 +6,9 @@
     <title>Document</title>
 </head>
 <body>
```
Comment (DevBigEazi) [inline index.html:8]: pls add a bg of green to the body.

**Review by DevBigEazi — COMMENTED** [review]
Pls kindly adjust and re-commit

File: index.html (hunk 1 above)
Comment (yusufroqib) [reply index.html]: Fixed

File: index.html (hunk 2 above)
Comment (yusufroqib) [reply index.html]: Fixed @DevBigEazi

File: index.html [hunk 3]
```diff
@@ -5,9 +5,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
 </head>
-<body>
+<body style="background-color: green;">
     <header>
-        <nav> Nav Bar Test </nav>
+        <nav> NAVBAR TEST </nav>
```
Comment (DevBigEazi) [inline index.html:10]: capitalize nav text. I aint talk about uppercase

**Review by DevBigEazi — CHANGES_REQUESTED** [review]
pls make changes asap

File: index.html (hunk 3 above)
Comment (yusufroqib) [reply index.html]: Alright. Fixed

**Review by DevBigEazi — APPROVED** [review]
ok nice

**Pull request merged**